apprise/plugins/__init__.py

```diff
@@ -518,6 +518,7 @@ def url_to_dict(url, secure_logging=True):
 
     # Ensure our schema is always in lower case
     schema = schema.group('schema').lower()
+
```

Comment:
I'm not sure this line/file even needs to be touched in your PR, no?

Reply:
You're right.